File: Prova_1/gamma.m
function img = gamma(img, c, epsolon)

    img = im2double(img);

    [rows, columns] = size(img);

    for i = 1: rows

        for j = 1: columns

            img(i, j) = c * img(i, j)^epsolon;

        end
    end
end

File: Prova_1/questao_2_python/transforms/gamma.py
from skimage.util import img_as_float


def gamma(img, c, g):
    img = img_as_float(img)

    rows, columns = img.shape

    for i in range(1, rows):
        for j in range(1, columns):
            img[i, j] = c * img[i, j] ** g

    return img
